core/src/main/codegen/templates/Parser.jj

```diff
@@ -3955,7 +3959,32 @@ SqlCreate SqlCreate() :
     }
 }
 </#if>
-
+<#if parser.execStatementParserMethods?size != 0>
+/**
+ * Parses an EXEC statement
+ */
+SqlNode SqlExec() :
+{
+    final Span s;
+	SqlNode macro;
```

Comment:
nit: indentation

Reply:
Odd, it looks fine locally.

Reply:
Could it be related to using tabs vs. spaces in your editor?

Reply:
Looks like it, should be resolved in next commit